homeassistant/components/axis/const.py

```diff
@@ -14,8 +14,10 @@
 CONF_CAMERA = "camera"
 CONF_EVENTS = "events"
 CONF_MODEL = "model"
+CONF_STREAM_PROFILE = "stream_profile"
 
 DEFAULT_EVENTS = True
+DEFAULT_STREAM_PROFILE = "Don't use a stream profile"
```

Comment:
Maybe use a shorter statement here as this won't be translatable? Eg `"No profile"`.

Reply:
I'll use no stream profile since I don't think its translated from Axis either